### DISCUSSION TITLE
Screen: One zone / Multi zone / Strong (Strix Scar 18 2024)

### DISCUSSION THREAD

**Discussion by george971986** [seerge/g-helper#3452] [Q&A] [answered]
Hello, 

I was struggling to understand a few things about one zone/ multi zone / strong feature. For me, it is very unconfortable to use anything except "one zone" and the problem is that on a fresh new windows install, "multi zone" is set by default.  Is this a hard coded thing in the motherboard/BIOS? What is it? I already tried every possible windows 11 / intel / nvidia settings related to brightness and power and none was able to change this to "one zone" except G-helper or Armoury Crate. For me it almost feels like Asus broke the colors gamut on purpose only to come later and fix it with armoury crate bloatware :)) Thanks God there is G-helper!

**Comment by seerge** [comment]
@george971986 hello,

G-Helper would just "read" and show currently active mini-led setting. If it resets back to "multi-zone" on each reboot - it means it's done like that on firmware level in the device.

**Reply by george971986** [reply]
Yes, I also believe it is done on firmware level and unfortunately every G-Helper/Armoury crate action is undone on a restart and not memorized on a firmware level. The interesting part is that some options such as keyboard and other lighting remains even on a new windows install (I tested only with armoury crate, not sure if G-helper does it to), so those changes are somehow memorized. I wish there was a way to make a custom bios update since Asus, even though they make decent hardware, their software is mediocre at best :(

**Reply by seerge** [reply, marked as answer]
@george971986 I actually might have a solution for you

Quit app, add following line to the beginning of `%appdata%\GHelper\config.json`, save it and start G-Helper again.

```
"force_miniled": 1,
```

With this param app would forcefully set last remembered mini-led setting on start.
